### PR TITLE
Update splash-screen.md

### PR DESCRIPTION
**Pull request by mickyarun**
Custom splash screen the documentation is not complete, stackoverflow many people are struggling to implement, so added missing method provideSplashScreen and link to github samples.

### CODE REVIEW

**Comment by mickyarun** [comment]
@googlebot I signed it!

**Comment by sfshaza2** [comment]
@xster, this PR adds one in-page example and links to other examples in a repo. It seems fine, but can you confirm?

File: src/docs/development/ui/advanced/splash-screen.md [hunk 1]
```diff
@@ -258,3 +273,4 @@ tips on how to build a [dynamic yet performant splash screen][].
 [pre-warming a `FlutterEngine`]: /docs/development/add-to-app/android/add-flutter-fragment#using-a-pre-warmed-flutterengine
 [must use an Xcode storyboard]: https://developer.apple.com/news/?id=03042020b
 [Human Interface Guidelines]: https://developer.apple.com/design/human-interface-guidelines/ios/visual-design/launch-screen/
+[Sample code]:https://github.com/flutter/flutter/tree/master/dev/integration_tests/android_splash_screens
```
Comment (sfshaza2) [inline src/docs/development/ui/advanced/splash-screen.md:276]:
```suggestion
[Sample code]: {{site.github}}/flutter/flutter/tree/master/dev/integration_tests/android_splash_screens
```

**Review by sfshaza2 — APPROVED** [review]
With the one change I made, it's good to go! LGTM

**Pull request merged**